google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/BulkMutation.java

```diff
@@ -114,6 +114,20 @@ public MutateRowsRequest toProto(RequestContext requestContext) {
         .build();
   }
 
+  /**
+   * Wraps the protobuf {@link MutateRowsRequest}.
+   *
+   * <p>WARNING: Please note that the project id & instance id in the table name will be overwritten
```

Comment:
just the table name

Reply:
Also please add a warning that the mutations should be idempotent (ie should not use server timestamps) because the mutations will be retried

Reply:
Also, lets mark this as BetaApi, in case this doesn't prove useful

Reply:
Thanks for the review, I have updated the JavaDoc with the same information. PTAL